src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapper.java

```diff
@@ -0,0 +1,55 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import lombok.extern.slf4j.Slf4j;
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.InstantType;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.UriType;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.model.fhir.PathologyRecord;
+
+import java.util.List;
+
+@Component
+@Slf4j
+public class BundleMapper {
+    private static final List<UriType> BUNDLE_META_PROFILE = List.of(new UriType("https://fhir.nhs.uk/STU3/StructureDefinition/ITK-Message-Bundle-1"));
+    private static final String BUNDLE_IDENTIFIER_SYSTEM = "https://tools.ietf.org/html/rfc4122";
+
+    public Bundle mapToBundle(PathologyRecord pathologyRecord) {
+        Bundle bundle = generateInitialPathologyBundle();
+
+        // TODO: fullUrl is optional. But what is the fullUrl property on the entry? E.g., "urn:uuid:a4409d7c-b613-477c-b623-87e60406c2f0"
+        pathologyRecord.getPerformers().forEach(
+            performer -> bundle.addEntry().setFullUrl("urn:uuid:some-unique-uuid").setResource(performer)
+        );
+
+        pathologyRecord.getRequesters().forEach(
+            requester -> bundle.addEntry().setFullUrl("urn:uuid:some-unique-uuid").setResource(requester)
+        );
+
+        pathologyRecord.getSpecimenCollectors().forEach(
+            specimenCollector -> bundle.addEntry().setFullUrl("urn:uuid:some-unique-uuid").setResource(specimenCollector)
+        );
```

Comment:
It seems like all the objects from the `PathologyRecord` will be added the same way here (`.setResource(resource)`, `.setResource(anotherResource)`, `.setResource(andAnotherResource)` etc etc), and we'll have a lot of repetitive code.

Should `PathologyRecord` just be a simple list of `Resource`s, and then when we pass it here, we do the below once?

```
pathologyRecord.getResources().forEach(
        resource -> bundle.addEntry().setFullUrl("urn:uuid:some-unique-uuid").setResource(resource)
    );
```

Keen to hear people's thoughts.

Reply:
I prefer to keep the `PathologyRecord`. We need to gather all the data together and that's what it does, and then it needs to be put into a form suitable for egress which is a `Bundle` but could have been something else. Otherwise it seems to me like we give the `BundleMapper` too many responsibilities.

Reply:
For me, the PathologyRecord class being a wrapper for a bundle of resources, is important. It provides visibility of the contents of a pathology record. It increases testability as you can easily inject a mock of a pathology record into your tests.

Reply:
👍 Yeah sounds good with me